blog/district-c-collaboration/index.md

```diff
@@ -0,0 +1,201 @@
+---
+title:
+  'Student insights - the Hitachi Energy and district C technical writing
+  initiative'
+authors: niewiarowski
+date: '2024-10-17'
+tags:
+  - 'bez-kategorii'
+  - 'dobre-praktyki'
+  - 'warsztat'
+  - 'praktyki'
+  - 'articles-in-english'
+coverImage: 'group_photo.jpg'
+---
+
+In 2023, the Hitachi Energy Technical Writing Team in Krakow participated in
+District C, a project run by the University of North Carolina Wilmington (UNCW)
+to inspire the next generation of energy professionals.
+
+<!--truncate-->
+
+District C lets students explore a real-life company\'s business operations.
+They learn about the unique aspects of an industry hands-on. In the end,
+students offer practical advice to the given problem during a pitch meeting,
+often their first experience presenting a business case to stakeholders. Read
+the story of the collaboration between District C and Hitachi Energy in Krakow,
+written from the perspective of Andrzej, a member of the Technical Writing Team.
```

Comment:
czy to tylko twoja perspektywa czy tez czesciowo calego zespolu, artykul brzmi jak troche to i to... ;)

Reply:
w sumie napisane z kazdej perspektywy, prawda jest ze zarowno ja jak i zespol nie mielismy rozbieznosci w patrzeniu na ten program. Moze wyrazenie typu "on behalf of the technical writing team" byloby lepsze. Przeczytam artykul jeszcze raz i usune zmiany perspektyw w tekscie gdzie z artykulu opisowego przechodze do pierwszej osoby bo to faktycznie moze zgrzytac.

Reply:
kwestie te troche uladzilem